README.md

```diff
@@ -632,6 +638,46 @@ Example ESLint configuration:
 }
 ```
 
+### [typescript/member-delimiter-style]
```

Comment:
When is this rule useful to have enabled at the same time as using Prettier?

Reply:
I added it since it follows a similar rationale to `vue/html-self-closing` — it works fine with Prettier when configured as specified.

Reply:
Ok, but that's not what this config tries to do. There are plenty of rules that work just fine if configured correctly. But even then they don't provide any value. They just take time to execute and run the risk of conflicting in the future.

Reply:
Removed 🔥